Objects/floatobject.c

```diff
@@ -1683,8 +1684,97 @@ typedef enum _py_float_format_type float_format_type;
 #define ieee_big_endian_format _py_float_format_ieee_big_endian
 #define ieee_little_endian_format _py_float_format_ieee_little_endian
 
-#define float_format (_PyRuntime.float_state.float_format)
-#define double_format (_PyRuntime.float_state.double_format)
+#ifdef __GNUC__
+__attribute__((always_inline))
+#endif
+static inline uint32_t
+byte_swap_uint32(uint32_t x) {
```

Comment:
Use `_Py_bswap32` instead. It does what you think it does (in `pycore_bitutils.h`).

Reply:
Didn't find this, thank you.